environments/staging/app-processes.yml

```diff
@@ -1,31 +1,39 @@
 formplayer_command_args: '-javaagent:/home/cchq/dd-java-agent.jar -Dsrc.main.java.org.javarosa.enableOpenTracing=true'
 management_commands:
-  celery3:
+  celery_a0:
     run_sms_queue:
     queue_schedule_instances:
     handle_survey_actions:
   pillow3:
     run_pillow_retry_queue:
 celery_processes:
-  'celery3':
-    repeat_record_queue,reminder_case_update_queue,reminder_rule_queue,export_download_queue,case_import_queue:
-      concurrency: 4
-    celery:
-      concurrency: 4
+  celery_a0:
     background_queue,case_rule_queue,analytics_queue,linked_domain_queue:
       concurrency: 2
+    repeat_record_queue,reminder_case_update_queue,reminder_rule_queue:
+      concurrency: 4
     async_restore_queue:
       concurrency: 1
+    case_import_queue:
+      concurrency: 4
+      prefetch_multiplier: 1
+    celery:
+      concurrency: 4
     celery_periodic:
       concurrency: 1
     email_queue:
       concurrency: 1
+    export_download_queue:
+      concurrency: 4
+      prefetch_multiplier: 1
+    malt_generation_queue:
+      concurrency: 1
     reminder_queue:
       pooling: gevent
       concurrency: 4
     saved_exports_queue:
       concurrency: 3
-      optimize: True
+      prefetch_multiplier: 1
```

Comment:
mildly curious what this entails? Though I trust you've added it for good reason.

Reply:
Regarding `optimize`, it's no longer in use, I'll make a PR to clean up the dead code.